docs/index.md

```diff
@@ -1,22 +1,32 @@
----
-layout: default
-title: Compose: Multi-container orchestration for Docker
 page_title: Compose: Multi-container orchestration for Docker
-page_description: Compose: Multi-container orchestration for Docker
-page_keywords: fig, composition, compose, docker
----
+page_description: Introduction and Overview of Compose
+page_keywords: documentation, docs,  docker, compose, orchestration, containers
 
-# <strong class="strapline">Fast, isolated development environments using Docker.</strong>
 
-Define your app's environment with a `Dockerfile` so it can be reproduced anywhere:
+## Overview
+
+Compose is a tool that allows you to orchestrate multiple Docker containers.
+With Compose, you can build clusters of containers which provide the resources
+(services, volumes, etc.) needed to build and run a complete distributed
+application.
+
+You can use Compose to build your app with containers hosted locally, or on a
+remote server, including cloud-based instances. Compose can also be used to
+deploy code to production.
```

Comment:
@aanand @SvenDowideit @fredlf Would you mind if I could get a LGTM on each docs change? At least until the 1.1.0 release, then we can figure out a better process.